src/pages/workspaces/workspace/workspace-utils.ts

```diff
@@ -0,0 +1,43 @@
+export type CloudProviderType = 'AZURE' | 'GCP'
```

Comment:
Since the workspace object is used all over the place, these types should definitely live at a higher level than pages/workspaces/workspace. @msilva-broad do you have a preference on where?

Reply:
agreed.  If they are data-centric (which these appear to be), then the best spot would be within src/libs/ajax.

Reply:
moved it to a new file, ajax-common.ts rather than throwing more stuff in the larger Ajax module which we are trying to move away from

Reply:
worth mentioning this suggestion does require changing ajax-common to a ts file...